source/macros.tex

```diff
@@ -217,12 +217,12 @@
 %% Notes and examples
 \newcommand{\noteintro}[1]{[\,\textit{#1:}\space}
 \newcommand{\noteoutro}[1]{\textit{\,---\,end #1}\,]}
-\newenvironment{note}[1][Note]{\noteintro{#1}}{\noteoutro{note}\xspace}
-\newenvironment{example}[1][Example]{\noteintro{#1}}{\noteoutro{example}\xspace}
+\newenvironment{note}[1][Note]{\noteintro{#1}}{\noteoutro{note}\space}
+\newenvironment{example}[1][Example]{\noteintro{#1}}{\noteoutro{example}\space}
 
 %% Library function descriptions
-\newcommand{\Fundescx}[1]{\textit{#1}\xspace}
-\newcommand{\Fundesc}[1]{\Fundescx{#1:}}
+\newcommand{\Fundescx}[1]{\textit{#1}}
```

Comment:
This is a separate issue, but is there any reason to keep `\Fundescx` around?

Reply:
It doesn't seem to be used, except for \Fundesc.